pulp_2to3_migration/app/migrations/0031_add_repoid_to_deb_types.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 2.2.20 on 2021-12-07 13:54
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pulp_2to3_migration', '0030_update_json_field'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='pulp2debcomponent',
+            name='repoid',
+            field=models.TextField(default='blub'),
```

Comment:
For now, I have not tried to implement anything to add a correct `repoid` to already created entries.
To be honest I feel like in this case with the migration plugin it makes more sense to drop all entries from the affected tables and then re-run the deb migration...

Just setting `default='blub'` is clearly not a real solution, but I thought I would ask if there is any experience with how to handle such cases before implementing something random. The current migration only works if it is applied before the relevant tables have any entries.

Reply:
I think it's fine to set repo_id to some string for the db migration sake.
You can't fix db data properly for users to just rerun a content migration (maybe in theory you can but it's difficult and error prone). They will need to run reset first anyway, so your random string in the repo_id will be cleaned up during reset.